wiki/Tournaments/OZT/2018/en.md

```diff
@@ -0,0 +1,114 @@
+# osu! Z-Tournament 2018
+![osu!Z-Tournament 2018](logo.png)
+
+- The **osu! Z-Tournament 2018**, is a 4v4 international osu!standard tournament.
+- There are no rank limits, therefore allowing any player to join.
+- Score system: **Scorev2**
+- This tournament will hold up to **64 teams, with 6 players at minimum and 8 players at maximum in each team.**
+- **Team admission will be decided through their PP average**, which means that the teams with the highest PP average will take priority.
+- **Group stages will be held round robin style**, afterwards the **Bracket stages will be double elimination**
+- It is **required** that all players join the discord
+- This page will only cover the general rules of the tournament, **you are required to read the ruleset.**
+- Any further questions must be forwarded to **[BearZyo](https://osu.ppy.sh/u/6116759) or [Div](https://osu.ppy.sh/u/3751116)**
+
+## Links
+- [Discord](https://discord.gg/29RmHDQ)
+- [Stream](https://www.twitch.tv/ztournament)
+- [Spreadsheet](https://docs.google.com/spreadsheets/d/e/2PACX-1vT24OMXj6iKV2XeAHADas4JtTGi1E_dAfxeiAMt4-j_smF62yWz92H03LUrLkLjltk9M6zHz7iQSHlk/pubhtml)
+- [Ruleset](https://docs.google.com/document/d/1JailsFw7ZMr9svB_gxxd0uMX2yCkhw7CY4VX4M4TTQs/edit?usp=sharing)
+- MapPool
+- [Forum Post](https://osu.ppy.sh/forum/t/735631)
+
+## Tournament Schedule
+
+| Event | Timestamp (YYYY-MM-DD) |
+| ---: | :--- |
+| Registration Phase | 2018-04-23 / 2018-05-25 |
+| Group Drawings | 2018-05-27 |
+| Group Stage Week 1 | 2018-06-02 / 2018-06-03 |
+| Group Stage Week 2 | 2018-06-09 / 2018-06-10 |
+| Round of 32 | 2018-06-16 / 2018-06-17 |
+| Round of 16 | 2018-06-23 / 2018-06-24 |
+| Quarter-Finals | 2018-06-30 / 2018-07-01 |
+| Semi-Finals | 2018-07-07 / 2018-07-08 |
+| Finals | 2018-07-14 / 2018-07-15 |
+| Grand Finals | 2018-07-21 / 2018-07-22 |
+
+## Prizes
+- The prizes are subject to change throughout the tournament.
+
+| Place | Prize |
+| ---: | :--- |
+| 1st | Profile Badge |
+| 2nd | |
+| 3rd | |
+
+## Groups
+- TBA: 2018-05-27
+
+## Brackets
+- TBA: 2018-06-11
+
+## Registrations
+**Register your team here!** Remember, all team members must join the discord for the team to be eligible to play.
+- [Team Registration](https://docs.google.com/forms/d/e/1FAIpQLSeNBB5MHAsdFCYxbHYR3v97OIHzCbUNRErOVYf023VAkGnGaQ/viewform)
+
+
```

Comment:
I guess you can add a horizontal bar in between the thing below and above
`---`